docs/index.md

```diff
@@ -44,138 +58,3 @@ provider "instaclustr" {
 ## Migrating from v1 to v2 of the Instaclustr Terraform Provider
 
 With the v2 version of the Instaclustr Terraform Provider, new resources have been introduced and schemas of existing resources have been changed. While the [Terraform Registry](https://registry.terraform.io/providers/instaclustr/instaclustr/latest/docs) contains the schema definitions of each resource, for a tool assisted approach of migrating to the v2 version of the Terraform Provider, see our support article on [importing Terraform resources](https://www.instaclustr.com/support/api-integrations/integrations/terraform-code-generation/).
-
-## Known Limitations
```

Comment:
Similar to above, would need to be added to `/docs-base/index.md` to have 0 differences in this file. Probably best to check with the team that rolled out this change first to confirm with them and let them know first.

Reply:
Same as above